discord/http.py

```diff
@@ -2185,6 +2185,12 @@ def move_member(
             guild_id=guild_id, user_id=user_id, channel_id=channel_id, reason=reason
         )
 
+    def set_voice_channel_status(
+        self, channel_id: Snowflake, payload: Any, *, reason: str | None = None
```

Comment:
I would revise this to take `status` as a string param and build the payload, similar to the other http methods